projectgenerator/gui/generate_project.py

```diff
@@ -181,3 +192,18 @@ def link_activated(self, link):
 
         else:
             QDesktopServices.openUrl(link)
+
+    def select_crs(self):
+        """ Open a dialog to select CRS """
+        crsSelector = QgsProjectionSelectionDialog(self)
+        crsSelector.setCrs(QgsCoordinateReferenceSystem(self.epsg, type=QgsCoordinateReferenceSystem.PostgisCrsId))
+        res = crsSelector.exec()
```

Comment:
Unlike with PyQt4, now both `exec()` and `exec_()` do the job. See [this answer](http://stackoverflow.com/questions/22610720/should-i-use-app-exec-or-app-exec-in-my-pyqt-application/22614643#22614643).